input/metadata/properties/default.json

```
@@ -4878,7 +4878,6 @@
     "field_size": {
       "type": "number",
       "description": "Size of the field.",
```

```suggestion
      "description": "Size of the field.",
      "units": "ha",
```
Restore the previous state

---

Confirmed change that should be happening in metadata properties - "units" is not a valid key and we shouldn't be tracking it in the metadata. Will document this though in the new PR.